client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/upgrade-modal.tsx

```diff
@@ -58,37 +58,21 @@ const UpgradeModal = ( { slug, isOpen, closeModal, checkout }: UpgradeModalProps
 			text: (
 				<p>
 					{ translate(
-						'You can purchase a subscription to use this theme or join the Premium plan to get it for free.'
+						"Gain access to all our Premium themes, and a ton of other features, with a subscription to the Premium plan. It's %s a year, risk-free with a 14-days money-back guarantee.",
```

Comment:
We need to refine this slightly -- we now have a handful of premium themes that require a Business plan (because they need to install WooCommerce on an Atomic site). Also, we use `14-day money-back guarantee` elsewhere in Calypso.

Also, I think we should bold the price to make it super-clear what the price is.

```suggestion
						"Get access to our Premium themes, and a ton of other features, with a subscription to the Premium plan. It's {{strong}}%s{{/strong}} a year, risk-free with a 14-day money-back guarantee.",
```

Reply:
![](https://user-images.githubusercontent.com/10071857/203934603-15418d56-70b8-46a7-bffa-41782cf9eb44.png)